client/galaxy/scripts/apps/login.js

```diff
@@ -0,0 +1,39 @@
+
+var jQuery = require( 'jquery' ),
+    $ = jQuery,
+    PANEL = require( 'layout/panel' ),
+    _l = require( 'utils/localization' ),
+    PAGE = require( 'layout/page' );
+
+window.app = function app( options, bootstrapped ){
```

Comment:
Loading everything at once would not happen in either pattern, right? We could just dynamically require each app, but either way, if the slight loss of convenience does not bother and the redundancies are gone, this looks good. I do like the 'tourability' aspect tho. It would be fantastic if we could somehow preserve/improve it. After all, tours have the potential to execute entire workshops as test cases.